### PR TITLE
Edit resources title + Information popup always visible

### PR DESCRIPTION
**Pull request by LaChope**

### CODE REVIEW

File: src/components/PlanningTool.jsx [hunk 1]
```diff
@@ -558,6 +582,84 @@ class PlanningTool extends Component {
...
+
+    const group = groups.find(g => g.id === id)
+    group.showIcons = true
+
+    this.setState({
+      groups: groups
+    })
+  }
+
+  handleShowIconsOnMouseLeave = (id) => {
```
[the remark this answers is not in the document]
Comment (blcham) [reply src/components/PlanningTool.jsx:596]: By the way, not sure as I did not read the code fully, but is it very clear from the context what is `id` ?  Wouldn't it better to call it `groupId`?

**Review by blcham — APPROVED** [review]
LGTM, my biggest concerns are about usage of `px`.

Notifying about impementation of the feature -- @kostobog

File: src/assets/PlanningTool.css [hunk 2]
```diff
@@ -26,9 +38,18 @@
...
+  padding: 2rem;
+  flex-direction: column;
+  top: 0;
+  right: 0;
+}
 
 .explanatory-notes, .popup {
-  margin: 25px;
+  background-color: #dadada;
+  margin-top: 25px;
```
[the remark this answers is not in the document]
Comment (LaChope) [reply src/assets/PlanningTool.css:52]: Done

File: src/components/PlanningTool.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (LaChope) [reply src/components/PlanningTool.jsx:596]: > how about reusing code by having method:
> 
> ```
> setIconVisibility(id, showIcon)
> ```

I tried with a single method with `group.showIcons = !group.showIcons`. However, it introduces some undesirable effects (e.g. icon not hiding when leaving edit mode and mouse still pointing at the input field)

[the remark this answers is not in the document]
Comment (LaChope) [reply src/components/PlanningTool.jsx:596]: > By the way, not sure as I did not read the code fully, but is it very clear from the context what is `id` ? Wouldn't it better to call it `groupId`?

Done.

**Pull request merged**